src/index.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Promises practice</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1 class="logo">MA</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+  <html lang="en">
```

Comment:
fix it
<img width="593" alt="image" src="https://user-images.githubusercontent.com/86183007/195557859-749f3fa7-8c59-4020-a184-bb1c43e16c57.png">